lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -208,8 +208,12 @@ function dosomething_global_form_node_form_alter(&$form, &$form_state, $form_id)
   if (!(in_array($form['type']['#value'], $translatable_types))) {
     return;
   }
+
   // Get the last item in the url, to get the edit/add node language.
   $node_lang = array_pop(explode('/', (parse_url(current_path(), PHP_URL_PATH))));
+  if ($node_lang == 'edit') {
```

Comment:
well we could but then we'd have to add an else statement under this new IF I created, feels like it's just making more code than we really need

is it confusing the way it is?